### PR TITLE
Mail template option

### PR DESCRIPTION
**Pull request by Adirelle**
Contribution Type: new feature
Primary Area: plugins
Link to Bug: #907

Description of change: 
* Add an "template" option for the email plugin to select the template to use (between "short" and "long").

Description of solution:
* Add a common layout for emails, with dynamic parts (color and introducing phrases).
* Rename the old "failed" and "success" templates to "short" and "long".
* For backwards compatibility, when the "template" option is missing, the selection is based on the build status: "short" for failed builds, "long" for successful builds.

### CODE REVIEW

File: PHPCI/Plugin/Email.php [hunk 1]
```diff
@@ -70,12 +70,21 @@ public function execute()
 
         $buildStatus  = $this->build->isSuccessful() ? "Passing Build" : "Failing Build";
         $projectName  = $this->build->getProject()->getTitle();
-        $mailTemplate = $this->build->isSuccessful() ? 'Email/success' : 'Email/failed';
+        $mailTemplate = $this->build->isSuccessful() ? 'Email/long' : 'Email/short';
```
Comment (tvbeek) [inline PHPCI/Plugin/Email.php]: Isn't it more logic to get a long email on failure and the short one on success? (Maybe that is just logic for me, I'm not sure)

File: PHPCI/Plugin/Email.php [hunk 2]
```diff
@@ -70,12 +70,21 @@ public function execute()
...
+
+        if (!empty($this->options['template'])) {
```
Comment (tvbeek) [inline PHPCI/Plugin/Email.php]: Is it possible to check if the template exist? (If I made a typo it can result in an error)

Comment (Adirelle) [inline PHPCI/Plugin/Email.php]: Not sure. I don't know if the b8 framework allows this.

File: PHPCI/Plugin/Email.php (hunk 1 above)
Comment (Adirelle) [inline PHPCI/Plugin/Email.php]: You're right and I agree. I just have kept the current behavior. Should we change that ?

**Comment by Adirelle** [comment]
@mavimo Changes done.

**Pull request closed without merging**